pypa_theme/static/sidebar.js

```diff
@@ -71,7 +72,7 @@ $(function() {
     bodywrapper.css('margin-left', bw_margin_collapsed);
     sidebarbutton.css({
         'margin-left': '0',
-        'height': bodywrapper.height(),
+        'height': documentwrapper.height(),
```

Comment:
The reason for making this change is: If there's an element with margin at an extreme, the sidebar didn't take 100% height of the white portions on load.

This fixes that issue.